src/claim/unique-id/unique-id.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+import { Entry } from '../entry/entry';
+
+const snarkjs = require('snarkjs');
+const utils = require('../../utils');
+const CONSTANTS = require('../../constants');
+
+const { bigInt } = snarkjs;
+const helpers = require('../../sparse-merkle-tree/sparse-merkle-tree-utils');
+
+/**
+ * Class representing a unique identifier claim
+ * Unique identifier is used to store an object representation through a hash
+ * Unique identifier entry representation is as follows:
+ * |element 3|: |empty|object type|hash type|version|claim type| - |12 bytes|4 bytes|4 bytes|4 bytes|8 bytes|
+ * |element 2|: |unique identifier| - |32 bytes|
+ * |element 1|: |empty|identity| - |12 bytes|20 bytes|
+ * |element 0|: |empty| - |32 bytes|
+ */
+export class UniqueId {
```

Comment:
There are some "object type"s that are not unique to a person, like address, givenName, dob, etc.  Maybe another name for this claim type would be more appropriate?
- PersonalId
- PersonalInfo
- PersonalDetail

Reply:
Claim is finally named `LinkObjectIdentity` :)